main.cpp

```diff
@@ -10,64 +10,136 @@ video: Chapter 2 - Part 3
 
  Create a branch named Part1
 
- Purpose:  This project will teach you how to declare variables and free functions.  
- This will be the first project where the code you write will be compiled and you will be responsible for making sure it compiles before submitting it for review.
+ Purpose:  This project will teach you how to declare variables and free functions.  
+ This will be the first project where the code you write will be compiled and you will be responsible for making sure it compiles before submitting it for review.
+ 
  
+ 1) Write down the names of the 6 major primitive types available in C++  here:
  
- 1) Write down the names of the 6 major primitive types available in C++  here:
+    1. Boolean type
+        - bool
+
+    2. Integer types
+        - short
+        - int
+        - long
+        - long long
  
+    3. Chracter types
+        - char
+        - wchar_t
+        - char16_t
+        - char32_t
+
+   4. Floating point types
+        - float
+        - double
+        - long double
+
+    5. Void type
+        - void
+
+    6. Null Pointer Type
+        - nullptr_t
```

Comment:
While this list is accurate, we don't need this many types when writing juce plugins. 
we only need:
```
int
float
double 
bool
char
void
```

Reply:
Thanks for information